SC Evo Complete/SCEvo_Core.SC2Mod/Base.SC2Data/GameData/SCBW/Protoss_Units.xml

```diff
@@ -4706,6 +4710,8 @@
         <DataRecord Entry="Turret,DragoonSCBW"/>
         <DataRecord Entry="Unit,DragoonSCBW"/>
         <TechInfoUpgradeUsed value="DragoonSCBWRangeUpgrade"/>
+        <UpgradeInfoWeapon index="0" DamagePerDice="1.5"/>
```

Comment:
refers to the medium damage, bonuses are done manually in old school upgrade effects